stream_alert/alert_processor/outputs.py

```diff
@@ -14,17 +14,24 @@
 limitations under the License.
 """
 from abc import abstractmethod
-import cgi
 from collections import OrderedDict
 from datetime import datetime
+import cgi
 import json
 import os
 import uuid
 
+from botocore.exceptions import ClientError
+import backoff
```

Comment:
This needs added to the `alert_processor_config` in `conf/lambda.json`

Reply:
oh right, it is in my internal config, i'll push it to the default